recipes/tip/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "tip" %}
+{% set version = "v1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IM-USAF/tip/archive/refs/tags/{{ version }}.tar.gz
```

```suggestion
{% set version = "1.0.2" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/IM-USAF/tip/archive/refs/tags/v{{ version }}.tar.gz
```
The conda version cannot have a `v`!